docs/developing/oauth.rst

```diff
@@ -22,14 +24,22 @@ Redirect URIs
 
 Store the Client ID and Client Secret tokens for use with your application.
 
-* These are the recommended settings. For a better understanding of which settings you should choose, read this [introduction to OAuth](https://aaronparecki.com/oauth-2-simplified/) and this [guide to grant types](https://alexbilbie.com/guide-to-oauth-2-grants/)
+* These are the recommended settings. For a better understanding of which settings you should choose, read this `introduction to OAuth <https://aaronparecki.com/oauth-2-simplified/>`_ and this `guide to grant types <https://alexbilbie.com/guide-to-oauth-2-grants/>`_
 
 Requesting authorization
 ========================
 
 Inside your application, redirect to the OAuth authorization endpoint to receive an authorization code. The URL is https://ion.tjhsst.edu/oauth/authorize/
 
-To access the API, exchange this code for a (temporary) access token. The URL is https://ion.tjhsst.edu/oauth/token/
+To access the API, exchange this code for a (temporary) access token. The URL is https://ion.tjhsst.edu/oauth/token/.
+
+Python-social-auth
+------------------
+
+If you want to use `python-social-auth`, a plugin is available in the ion_oauth package.
+See `ion_oauth <https://pypi.python.org/pypi/ion_oauth>`_
```

Comment:
Note that the package is not currently actively maintained by the current Ion dev team (though may in the future depending on whether they have the time).